ReactJS-Project-CardinalKit/src/components/LoginPage.jsx

```diff
@@ -3,58 +3,194 @@ import * as PropTypes from 'prop-types';
 import { Redirect } from 'react-router-dom';
 
 import { connect } from 'react-redux';
-import { defineMessages, FormattedMessage } from 'react-intl';
 
 import { loginUser } from '../actions/loginActions';
 import { getLoginState, isAuthenticated } from '../selectors/loginSelectors';
+import app from 'firebase/app';
 
-import { Button, ButtonColor, ButtonType } from '../ui/Button';
 
-import logo from '../images/cardinal_logo.svg';
+import { Button, ButtonColor, ButtonType } from '../ui/Button';
+import Firebase from './Firebase';
+import logo from '../images/login-office.jpeg';
+import logo2 from '../images/cardinal_logo.svg';
 
-const messages = defineMessages({
-  header: {
-    id: 'LoginPage.header',
-    defaultMessage: 'Login with Google',
-  },
-});
+// const messages = defineMessages({
+//   header: {
+//     id: 'LoginPage.header',
+//     defaultMessage: 'Login with Google',
+//   },
+// });
 
 export class LoginPage extends React.Component {
+  constructor(props) {
```

Comment:
No store actions seems to be implemented here. If not usable, remove the loginActions from the store,

Reply:
@vjbhaskar-ui we noted down this point previously they used Store, We will check and remove this.